packages/amplify-dynamodb-simulator/index.js

```diff
@@ -229,12 +229,14 @@ async function launch(givenOptions = {}, retry = 0, startTime = Date.now()) {
 }
 
 function getClient(emu, options = {}) {
-  const { DynamoDB } = require('aws-sdk');
+  const { DynamoDB } = require('@aws-sdk/client-dynamodb');
```

Comment:
Strange ... I could have sworn there was more of a spiderweb of changes related to this when I tried this change in the parent branch. Will be interesting to see what E2Es turn up ...

Reply:
Anticipating this will be uneventful at this point, but running e2es again:

https://us-east-1.console.aws.amazon.com/codesuite/codebuild/594813022831/projects/amplify-category-api-e2e-workflow/batch/amplify-category-api-e2e-workflow:67a57a07-356e-4e63-a01f-f49bdcc7cdee?region=us-east-1